schema.sql

```diff
@@ -8,4 +10,44 @@ create table animals(
     neutered bool not null,
     weight_kg decimal not null,
     primary key (id)
-);
\ No newline at end of file
+);
+
+-- project1
+
+alter table animals
+add column species text;
+
+begin;
+savepoint sp0;
+update animals
+set species = 'unspecified';
+select * from animals;
+rollback to sp0;
+
+begin;
+update animals
+set species = 'digimon'
+where name like '%mon';
+update animals
+set species = 'pokemon'
+where species is null;
+commit;
+select * from animals;
+
+begin;
+delete from animals;
+rollback;
+select * from animals;
+
+begin;
+delete from animals
+where date_of_birth > '2022-01-01';
+savepoint sp0;
+update animals
+set weight_kg = weight_kg * (-1);
+rollback to sp0;
+update animals
+set weight_kg = weight_kg * (-1)
+where weight_kg < 0;
+commit;
+select * from animals;
```

Comment:
- [ ] Make sure Everything that is a query that helps to analyze data should be saved in the `queries.sql` file and not in the `schema.sql` file, Kindly remove it from here that can be only written in the `queries.sql` file 👍🏻